ports/hareflow/portfile.cmake

```diff
@@ -0,0 +1,21 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO coveooss/hareflow
+    REF f5a0b08e08a1b69aed11ae39574cc02cfd06a855 # 0.1.0
+    SHA512 4d0b432ba1d335ddad4ac750b479441ed5cea5f55ff3e38f09e39ebea76c676ebae10dbc9dc2d375cb11d550bf2ae67c4383865edcdc0adbc3dca52a94d276ea
+    HEAD_REF main
+)
+
+if(VCPKG_TARGET_IS_OSX OR VCPKG_TARGET_IS_IOS)
+    set(rpath "@loader_path")
+else()
+    set(rpath "\$ORIGIN")
+endif()

```

Comment:
You set the variable `rpath`, but you dont use it anywhere. You should use it in `vcpkg_cmake_configure`.

Reply:
oups merci